src/docs/patterns/internationalization.mdx

```diff
@@ -1,54 +1,145 @@
+
+> **NOTE:** PX Blue recommends using [Noto Sans](https://www.google.com/get/noto/#sans-lgc) for all non-western languages. You can read more on our [Typography](/style/typography) page.
+
+## Bi-directionality/Right-to-Left (RTL) Support
 
 <MaterialDesignDescription
     title={'Bidirectionality'}
     description={'Learn about mirroring layout, mirroring elements, and localization.'}
     url={'https://material.io/design/usability/bidirectionality.html'}
 />
 
-## Technical Implementation
+Some languages, such as Arabic and Hebrew, are read from right to left. Applications supporting these languages will need to mirror the UI so the natural flow is from right to left. For example, navigation drawers should be displayed on the right side of the screen, icons should be to the right of the text in buttons, and any icons with directionality should be mirrored.
+
+<ImageGrid
+    images={[UIFlippedImage]}
+    regularWidth
+    caption={[
+        '1: A PX Blue user interface in right-to-left language (English).',
+        '2: The same PX Blue user interface in left-to-right language (Arabic).',
+    ]}
+/>
+<ImageGrid
+    images={[IconsFlippedImage]}
+    regularWidth
+    caption={[
+        '1: Icons that indicate directions should be mirrored.',
+        '2: Icons with glyphs and symbols should never be mirrored.',
+    ]}
+/>
+
+## Cultural Differences
+
+<ImageGrid
+    images={[CulturalDifferencesImage]}
+    regularWidth
+    caption={'People from different cultures are appealed to different things.'}
+/>
+
+Addressing the cultural differences is the key concern of the localization. Sometimes certain UI elements convey different meaning in different cultures. For example, in America, the red color in the stock market indicates a decline, while in China the same color indicates a rise. A thumb-up gesture usually expresses an approval in English-speaking country, yet the gesture can have a negative meaning in Iraq. When designing for a new market, you are encouraged to have a local consultant run an audit for your application.
+
+When you are designing for webpages (instead of applications), your primary goal is to advertise your products and services, and appeal to the local market. The language tone, page style, and choice of image assets are expected to be tailored for different targeted culture. Likewise, your user study and market research conclusions derived under a certain cultural background might not apply to another culture.
+
+<React.Fragment>
+    <MaterialDesignDescription
+        title={'Design Localization and the New Web'}
+        description={'An examination of localization in web design by Bitmatica Lab.'}
+        url={'https://medium.com/bitmatica-lab/design-localization-and-the-new-web-f54f43c5f040#'}
+        avatar={<Web style={{ width: 48, height: 48 }} />}
```

Comment:
Not a huge fan of this Web icon for these.
What about 
![image](https://user-images.githubusercontent.com/29152776/92471629-3ee0b380-f1a6-11ea-8226-421c2c4d1ea4.png)
or
![image](https://user-images.githubusercontent.com/29152776/92471659-4acc7580-f1a6-11ea-9006-85db4b3b04e9.png)

Reply:
I am using the link icon right now... feels a bit weird